toolkit/tools/bldtracker/bldtracker.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Tool to initialize a csv file or record a new timestamp
```

Comment:
```suggestion
// Tool to initialize a CSV file or record a new timestamp
```

Please fix other places in the comments as well, if there are more.